src/main/java/com/massivedisaster/bintraydeployautomator/model/Configuration.java

```diff
@@ -100,6 +113,23 @@ public void setBintrayKey(String bintrayKey) {
         return extraTasks.toArray(new String[0]);
     }
 
+    private String[] getBuildTasks() {
+        if (modules == null) {
```

Comment:
I don't really get why is this necessary.
modules should never been null. But if they are we aren't suppose to get here.

Reply:
it can be null, we defined in the beginning that if there is no modules run the script for entire gradle project.